src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -819,14 +819,14 @@
                       "timeInWeek": [
                         {
                           "day": "Tuesday",
-                          "hour_slots": [
+                          "hourSlots": [
```

Comment:
This might be a breaking change.

Reply:
Yes, this is a breaking change. I assume it's fine for a cli-extension of a preview feature.

Reply:
The reason we made this change is that we use "hourSlots" in REST API. Difference between cli and rest api is confusing the users.